### PR TITLE
Fix missing command

### PR DESCRIPTION
**Pull request by raphmur**
Merging to stage
@baylanger , please advise which one is relevant

### CODE REVIEW

File: subroutines.sh [hunk 1]
```diff
@@ -208,6 +208,18 @@ send_key() {
   done
 }
 
+scan_bleln_macaddr(){
+  # copied from legacy "scan_bluetooth" function. To decide if still relevant
+  # note there is this PR https://github.com/tesla-local-control/tesla-local-control-addon/pull/32
+  # quite old, but has the principles for auto populating the BLE MAC Address with only the VIN
+  vin=$1
+  VIN_HASH=$(echo -n ${vin} | sha1sum)
```
Comment (baylanger) [inline subroutines.sh:216]: We have a function for this... `BLE_ADVERT=$(tesla_vin2ble_ln $vin)`

Comment (raphmur) [reply subroutines.sh:216]: oh ok I see. I will update

**Comment by baylanger** [comment]
Added in PR 68, ended up with this :

```
scan_bleln_macaddr() {
  # copied from legacy "scan_bluetooth" function. To decide if still relevant
  # note there is this PR https://github.com/tesla-local-control/tesla-local-control-addon/pull/32
  # quite old, but has the principles for auto populating the BLE MAC Address with only the VIN
  vin=$1
  log_notice "Calculating BLE Local Name for vin:$vin"
  ble_ln=$(tesla_vin2ble_ln $vin)

  log_info "Looking in BLE cache for a device matching ble_ln:$bleln"
  if ! bluetoothctl --timeout 2 devices | grep $ble_ln; then
    log_debug "Did not find ble_ln:$ble_ln in the BLE cache"
    # Look for a BLE adverstisement matching ble_ln
    log_notice "Scanning (10 seconds) BLE advertisement for ble_ln:$ble_ln"
    bluetoothctl --timeout 10 scan on | grep $ble_ln
    [ $? -ne 0 ] && return 1
  fi
  log_debug "scan_bleln_macaddr; found MAC addr for vin:$vin ble_ln:$ble_ln"
}
```

**Comment by raphmur** [comment]
ok thanks.
FYI to change the branch a PR is merged into:
Click on EDIT on top
image

Then you can change branch (for an open PR)

**Pull request closed without merging**